assets/sass/components/dashboard/_googlesitekit-welcome-modal.scss

```diff
@@ -0,0 +1,92 @@
+/**
+ * Welcome modal styles.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-welcome-modal {
+		.mdc-dialog__surface {
+			overflow: hidden;
+		}
+
+		.mdc-dialog__container {
+			max-width: 491px;
+		}
+	}
+
+	.googlesitekit-welcome-modal__graphic {
+		background-image: linear-gradient(to bottom, $c-site-kit-sk-50, $c-white);
+		max-height: 200px;
```

Comment:
This is very nitpicky, but I feel we should add a `position: relative` here to contain the close button in this container.

Without:
<img width="866" height="230" alt="Image" src="https://github.com/user-attachments/assets/a06610b9-2a70-4ae4-9480-bf9a545dd065" />

With:
<img width="869" height="242" alt="Image" src="https://github.com/user-attachments/assets/bdd98cfe-81de-45f9-9cad-a6044e70a2b9" />

```suggestion
	.googlesitekit-welcome-modal__graphic {
		background-image: linear-gradient(to bottom, $c-site-kit-sk-50, $c-white);
		max-height: 200px;
		position: relative;
```